src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy

```diff
@@ -135,9 +138,6 @@ class ConnectedDevicesSpec extends HealthCheckSpecification {
             validateSwitchHasNoFlowRulesAndMeters(flow.destination.switchId)
         }
 
-        cleanup: "Restore initial switch properties"
-        srcLldpData && [data.switchPair.src, data.switchPair.dst].each { database.removeConnectedDevices(it.dpId) }
```

Comment:
Do we have cleanup action for this removeConnectedDevices(line:139) and for some tests below?
Something like this
`cleanupManager.addAction(OTHER, {database.removeConnectedDevices(flow.source.switchId)})`

Reply:
It's inside the method `getFlowWithConnectedDevices()`